easybuild/framework/easyconfig/default.py

```diff
@@ -88,6 +88,7 @@
     'easyblock': [None, "EasyBlock to use for building; if set to None, an easyblock is selected "
                         "based on the software name", BUILD],
     'easybuild_version': [None, "EasyBuild-version this spec-file was written for", BUILD],
+    'github_account': [None, "Github account name in case of github source_urls", BUILD],
```

Comment:
make it clear this is used to resolve a template:

`"GitHub account name to be used to resolve template values in source URLs"`